librz/include/rz_analysis.h

```diff
@@ -807,6 +809,13 @@ typedef enum rz_analysis_data_type_t {
 	RZ_ANALYSIS_DATATYPE_FLOAT,
 } RzAnalysisDataType;
 
+// For switchting to AST-like approach in the future
+typedef struct rz_analysis_rzil_op_t {
+	RzPVector *ops;
+	// TODO : use root approach in the future
+	RzILOp **root_node;
+} RzAnalysisRzilOp;
+
```

Comment:
This is just a comment:
After we merge this, we should move esil and rz_il to it's own headers.

Reply:
This comment should be removed then, since it's already root approach.